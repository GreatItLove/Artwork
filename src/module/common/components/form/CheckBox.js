import React from 'react';
import Checkbox from 'material-ui/Checkbox';

const CheckBox = ({ input, label, iconStyle }) => (
  <Checkbox
    label={label}
    checked={input.value ? true : false}
    onCheck={input.onChange}
    iconStyle={iconStyle}
  />
);

export default CheckBox;
