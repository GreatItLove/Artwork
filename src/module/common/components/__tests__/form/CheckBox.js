import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import CheckBox from '../../form/CheckBox';

describe('Checkbox', () => {
  it('Snapshot of Checkbox', () => {
    const meta = {};
    const input = {};
    const wrapper = shallow(<CheckBox {...{ meta, input }} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
