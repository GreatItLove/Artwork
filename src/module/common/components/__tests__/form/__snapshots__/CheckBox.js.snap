// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox Snapshot of Checkbox 1`] = `
<Checkbox
  checked={false}
  disabled={false}
  labelPosition="right"
/>
`;
